### PR TITLE
matter: samples: thermostat: doc: Fixed thermostat documentation

### PR DESCRIPTION
**Pull request by LipinskiPNordicSemi**
This commit fix documentation of thermostat

### CODE REVIEW

File: samples/matter/thermostat/README.rst [hunk 1]
```diff
@@ -272,17 +275,14 @@ After building this sample and the :ref:`Matter weather station <matter_weather_
 
    .. code-block:: console
 
-      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, \
-      {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": \
-      [{"cluster": 1026, "endpoint": <thermostat_sensor_endpoint_ID>, "deviceType": null}]}]' <Sensor Device NodeId> 0
+      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": [{"cluster": 1026, "endpoint": <temperature_sensor_endpoint_ID>, "deviceType": null}]}]' <Sensor Device NodeId> 0
```
Comment (kkasperczyk-no) [inline samples/matter/thermostat/README.rst:278]: Could we change `<Sensor Device NodeId>` to `<temperature_sensor_node_ID>` or something similar to follow the convention? Also could we mention this parameter in the bullet point description?

Comment (LipinskiPNordicSemi) [reply samples/matter/thermostat/README.rst]: good point

Comment (kkasperczyk-no) [reply samples/matter/thermostat/README.rst]: Btw. I meant also adding `<temperature_sensor_node_ID>` to the following text: 

> Use the following command, with *<thermostat_node_ID>* and *<temperature_sensor_endpoint_ID>* values from the previous step about commissioning:

**Review by Damian-Nordic — APPROVED** [review]

File: samples/matter/thermostat/README.rst [hunk 2]
```diff
@@ -257,6 +257,9 @@ The device reboots after all its settings are erased.
 Testing with external sensor
 ----------------------------
 
+.. note::
+   To enable external sensor option set :kconfig:option:`CONFIG_THERMOSTAT_EXTERNAL_SENSOR` to ``y`` in the configuration files or use the ``-DCONFIG_THERMOSTAT_EXTERNAL_SENSOR=y`` :ref:`CMake option <cmake_options>` when building the sample.
```
Comment (wiba-nordic) [inline samples/matter/thermostat/README.rst:261]:
```suggestion
   To enable the external sensor option, set the :kconfig:option:`CONFIG_THERMOSTAT_EXTERNAL_SENSOR` Kconfig option to ``y`` in the configuration files or use the ``-DCONFIG_THERMOSTAT_EXTERNAL_SENSOR=y`` :ref:`CMake option <cmake_options>` when building the sample.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

File: samples/matter/thermostat/README.rst [hunk 3]
```diff
@@ -272,17 +275,14 @@ After building this sample and the :ref:`Matter weather station <matter_weather_
...
    .. code-block:: console
 
-      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, \
-      {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": \
-      [{"cluster": 1026, "endpoint": <thermostat_sensor_endpoint_ID>, "deviceType": null}]}]' <Sensor Device NodeId> 0
+      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": [{"cluster": 1026, "endpoint": <temperature_sensor_endpoint_ID>, "deviceType": null}]}]' <temperature_sensor_node_ID> 0
 
 #. Write a binding table to the thermostat to inform the device about the temperature sensor endpoint.
-   Use the following command, with *<temperature_sensor_node_ID>* and *<thermostat_sensor_endpoint_ID>* values from the previous step about commissioning:
+   Use the following command, with *<temperature_sensor_node_ID>*, *<thermostat_sensor_endpoint_ID>* and *<temperature_sensor_node_ID>* values from the previous step about commissioning:
```
Comment (kkasperczyk-no) [inline samples/matter/thermostat/README.rst:281]: Please read this bullet point and the above one again to make sure the parameter names match the ones used in the command. For example, here you mention two times the same parameter `temperature_sensor_node_ID` and `thermostat_sensor_endpoint_ID` which is not present in the command. On the other hand we have in command `temperature_sensor_endpoint_ID` and `temperature_sensor_endpoint_ID` that are not described in the description.

For the bullet point above (regarding accesscontrol write acl) the `temperature_sensor_node_ID` is missing in the description.

**Review by kkasperczyk-no — CHANGES_REQUESTED** [review]

**Review by kkasperczyk-no — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]

**Pull request merged**